Gemfile

```diff
@@ -6,3 +6,4 @@ gem 'bundler-audit', '~> 0.9', require: false
 gem 'rake', '>= 12.3.3', require: false
 gem 'rubocop', '~> 1.51', require: false
 gem 'rubocop-rake', '~> 0.6', require: false
+gem 'toml', '~> 0.3', require: false
```

Comment:
This dep is:

- https://github.com/jm/toml
- https://rubygems.org/gems/toml/versions/0.3.0

Plenty of downloads on RubyGems with 18M+. Not sure how maintained it is: the last commit was in December 2021.

Reply:
Ah, you know what, I've found a better one (in all regards): https://github.com/fbernier/tomlrb

Better in that:
- No additional dependencies
- It's had more love over the years
- It explicitly supports TOML v1.0.0
- It's benchmarked itself against the only other compliant TOML library (toml-rb)

Also, it's MIT (same as `toml`, but just calling it out)

Reply:
Awesome. Change in the playground PR is at artichoke/playground@0070a9aecce16a581dc2090737bf7ae868b17fbc.